scripts/build.py

```diff
@@ -1382,93 +1382,9 @@ def shell(self):
 
 
     def release(self):
-        page = """
```

Comment:
I've always been annoyed by the fact that we inline html here. Sure it might be a bit easier to see the connection between the Mustache model we create in the release() function and the generated html but the build.py script is too long as it is.